bin/reth/src/cli/mod.rs

```diff
@@ -158,6 +158,17 @@ pub struct Logs {
     /// The filter to use for logs written to the log file.
     #[arg(long = "log.filter", value_name = "FILTER", global = true, default_value = "error")]
     filter: String,
+
+    /// Sets whether or not the formatter emits ANSI terminal escape codes for colors and other
+    /// text formatting.
+    #[arg(
+        long = "color", 
+        value_name = "COLOR",
+        value_parser = PossibleValuesParser::new(["always", "auto", "never"]), 
+        global = true,
+        default_value = "always"
+    )]
+    color: String,
```

Comment:
nit: lets make this an enum with a Fromstr impl

Reply:
ok